src/bootchooser.c

```diff
@@ -152,16 +155,16 @@ static gboolean barebox_set_primary(RaucSlot *slot) {
 			continue;
 
 		if (s == slot) {
-			prio = prio1;
+			prio = BAREBOX_STATE_NEW_PRIORITY;
```

Comment:
I would prefer something like `BAREBOX_STATE_PRIORITY_PRIMARY` as it better reflects the purpose of this value.